pkg/api/main.go

```diff
@@ -33,6 +36,13 @@ func Bitly(link string) {
 	if err != nil {
 		log.Fatalf("Error on response")
 	}
+
 	body, _ := ioutil.ReadAll(resp.Body)
-	log.Println(string([]byte(body)))
+	err = json.Unmarshal(body, &response)
+	if err != nil {
```

Comment:
terminate early if an error is caught?

Reply:
Yeah, terminate if it had issues unmarshalling the data